runtime/etc/Makefile.comm-ofi

```diff
@@ -33,7 +33,7 @@ LIBS += -lfabric -ldl
 #
 LD = $(CXX)
 
-ifneq (, $(findstring mpi,$(CHPL_MAKE_LAUNCHER)))
+ifneq (, $(findstring mpi, $(CHPL_MAKE_LAUNCHER) $(CHPL_COMM_OFI_OOB)))
```

Comment:
What's the behavior if `CHPL_COMM_OFI_OOB` is sockets and `CHPL_MAKE_LAUNCHER` contains mpi? Seems like `CHPL_COMM_OFI_OOB` should have precedence?

Reply:
> What's the behavior if `CHPL_COMM_OFI_OOB` is sockets and `CHPL_MAKE_LAUNCHER` contains mpi?

All that would happen in that case is that we'd search libmpi without linking anything from it, because there would be no references to MPI functions.  Really all this is doing is making sure we can find the MPI include files and libs if we need them, not checking whether or not we do need them.

Reply:
Oh, duh. I apparently only read the condition without actually looking at what was done when it was true.